robots/commenter/main.go

```diff
@@ -55,14 +59,16 @@ const (
 		.Issue.Assignees - list of assigned .Users
 		.Issue.Labels - list of applied labels (.Name)
 `
+	querySeparator = ","
 )
 
 func flagOptions() options {
 	o := options{
 		endpoint: flagutil.NewStrings(github.DefaultAPIEndpoint),
 	}
-	flag.StringVar(&o.query, "query", "", "See https://help.github.com/articles/searching-issues-and-pull-requests/")
-	flag.DurationVar(&o.updated, "updated", 2*time.Hour, "Filter to issues unmodified for at least this long if set")
+	flag.StringVar(&o.query, "query", "", "Comma-separated list of queries, see https://help.github.com/articles/searching-issues-and-pull-requests/")
```

Comment:
Should we name this `queries`?

Reply:
I suggest to not change the flags as we want to create an upstream PR

Reply:
Wouldn't it be like:
* keep it => keep the diff in the fork small
* change it => so that it's upstreamable
(of course only one of those)

Reply:
but if we change the flag every usage of the commenter has to be modified. I don't know if that's acceptable :man_shrugging:

Reply:
Tbh if you have only one query the flag is wrong. It could be used for queries, but is not intended. I would suggest to leave it as is

Reply:
Ah didn't know that it was named like this initially. Ok, then let's keep it as is it is.

Reply:
@tobiasgiese I just meant that the logic `keep as is because we want to create a upstream PR` doesn't really make sense :)

Reply:
Changed it... `queries` is okay for me